final/design/eval-strat.tex

```diff
@@ -3,29 +3,33 @@ \section{Evaluation Strategies}
 As explained in \cref{sec:supp-diff-ways}, there are multiple ways for
 specifying the dynamic semantics of a language. Interpretation can for example
 be performed through an interpreter generated from a DynSem specification, or by
-implementing the interpreter in Stratego or Java. However, in the end all of
-these methods do the same thing, that is they accept an AST as input and somehow
-transform it into an output. In Spoofax, both the input and the output can be
-represented as a Stratego term. It is only the internal representation of the
-evaluation context and the implementation of the interpreter that is different.
+implementing the interpreter in Stratego or Java. However, ultimately all of
+these methods do the same thing. That is, they accept an AST as input and
+somehow transform it into an output. In Spoofax, both the AST input and the
+output value can be represented as a Stratego term. This is due to the fact that
+Stratego supports all of the basic types that Java has: integers, real numbers,
+strings, lists and constructors for encoding objects. It is only the internal
+representation of an interpreter's evaluation context and the implementation of
+the interpreter that differs.
+% TODO: need reference for statement about Stratego terms?
```

Comment:
If you can find one it would be nice, but you shouldn't lose too much time searching for one.